src/frontend/apps/standalone_site/src/features/Contents/features/ClassRoom/components/ClassRoomRouter.tsx

```diff
@@ -12,7 +12,7 @@ const ClassRoomRouter = () => {
   const searchParams = new URLSearchParams(search);
   const playlistId = searchParams.get('playlist') || '';
 
-  const classroomRoute = routes.CONTENTS.subRoutes.CLASSROOM;
+  const classroomRoute = routes.CLASSROOM;
   const classroomCreatePath = classroomRoute.subRoutes?.CREATE?.path || '';
```

Comment:
As you removed the last `?` it in the `useAuthenticator`, is it possible to remove them here ?

```suggestion
const classroomCreatePath = classroomRoute.subRoutes?.CREATE.path || '';
```

Reply:
[fixup! ♻️(website) better routes separation for classroom](https://github.com/openfun/marsha/pull/2253/commits/01e2a2a6b10e7037353cad4b03940be5454e1c89)